### ISSUE TITLE
[Bug]: Syntax error in kubejs orechid

### ISSUE THREAD

**Issue by gxam** [AllTheMods/Spellbound#74]
### Possible Fixes

Yes

### Modpack Version

other

### What happened?

ATMS 2.2.2.

```
[10Jan2022 13:59:59.851] [Worker-Main-25/ERROR] [botania/]: Exception parsing orechid weights from KubeJS Resource Pack [data] / botania:orechid_ore_weights/orechid.json
```

This seems to be caused by yellorite missing its weight at line 116 in kubejs/data/botania/orechid_ore_weights/orechid.json:

```
    {
      "type": "tag",
      "tag": "forge:ores/yellorite",
    },
```

### latest.log

https://paste.ee/p/zjJtX

**Comment by MadWomen** [comment]
fixed in the next version.

**Comment by Mitchell5200** [comment]
Confirmed fixed in V2.2.7, Closing
